docs/authentication.md

```diff
@@ -60,6 +60,34 @@ public class Example {
 }
 ```
 
+### Authentication using private key and certificate content
+Instead of configuring the path of private key or certificate in config file, you can also authenticate the bot and 
```

Comment:
Maybe we could also mention that this is useful when either RSA private key or certificate are fetched from an external secrets storage